tests/Integration/Monolog/MessengerLoggingMiddlewareTest.php

```diff
@@ -22,21 +22,21 @@
 
 class MessengerLoggingMiddlewareTest extends TestCase
 {
-    public function testNoExceptionOccurred()
+    public function testNoExceptionOccurred(): void
     {
         $message = new TestMessage('test');
         $envelope = new Envelope($message);
 
         $loggingStrategy = $this->createMock(LoggingStrategyInterface::class);
-        $loggingStrategy->expects($this->never())
+        $loggingStrategy->expects(self::never())
```

Comment:
Why static methods ?

Reply:
We read this [post](https://discourse.laminas.dev/t/this-assert-vs-self-assert/448)  -> discussed in a team and then decided to use `self ::`